### DISCUSSION TITLE
Issue while installing Chevereto 4.0.5

### DISCUSSION THREAD

**Discussion by prbt2016** [chevereto/chevereto#5] [Q&A]
Hello,

I was in the process of installing Chevereto on Centos 7 with Apache 2.2, MYSQL 5.7 , PHP 8.0.22 using the release package via method mentioned here : 

https://v4-docs.chevereto.com/application/installing/installation.html#using-release-package

However this doesn't seem to work. On hitting the installation URL http://{{DOMAIN.COM/chevereto/install as mentioned in the docs . 

However the following is thrown : 

Something went wrong as shown in the screenshot. 

![image](https://user-images.githubusercontent.com/17871448/208440237-ecee09c8-2ea8-4458-9864-45c9dd944668.png)

How do I debug the same ?. 

Is  there any way to print the debug information on screen or under apache logs . 

Under apache logs nothing is thrown .

**Comment by rodber** [comment]
Refer to this message: https://chevereto.com/community/threads/re-something-went-wrong.14550/
(You need to debug to determine the actual error).
